source/ed/build_helper.cpp

```diff
@@ -31,6 +31,7 @@ www.navitia.io
 #include "build_helper.h"
 
 #include "ed/connectors/gtfs_parser.h"
+#include "utils/map_find.h"
```

Comment:
the famous :wink:

Reply:
the unfamous `map_find` :D 
It's now been removed because of its uselessness. (god that's a lot of `s` in this word....)